llvm/lib/SYCLLowerIR/SYCLPropagateAspectsUsage.cpp

```diff
@@ -665,6 +665,24 @@ void processDeclaredVirtualFunctionSets(
     StringMap<SmallVector<Function *, 4>> &VirtualFunctionSets) {
   if (!F->hasFnAttribute("calls-indirectly"))
     return;
+
+  // "Construction" kernels which reference vtables but do not actually
+  // perform any virtual calls have the calls-indirectly attribute
+  // attached by SYCLVirtualFunctionAnalysis pass. We do not want to
+  // attach sycl_used_aspects metadata to such kernels.
+  bool hasVirtualCall = false;
```

Comment:
```suggestion
  bool HasVirtualCall = false;
```

[LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly)